samples/modules/tflite-micro/hello_world/prj.conf

```diff
@@ -17,3 +17,4 @@ CONFIG_LIB_CPLUSPLUS=y
 CONFIG_NEWLIB_LIBC=y
 CONFIG_TENSORFLOW_LITE_MICRO=y
 CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_TENSORFLOW_LITE_MICRO_CMSIS_NN_KERNELS=y
```

Comment:
Since this sample can run on a target other than Cortex-M, this should not be added to the common configs.

Also note that the `sample.yaml` currently restricts this sample to `qemu_x86` and `qemu_x86_64`:
https://github.com/zephyrproject-rtos/zephyr/blob/fe7e6069a49e4ddbf0d31d299a8c41714bee6904/samples/modules/tflite-micro/hello_world/sample.yaml#L17

I would suggest adding a separate "test" for the CMSIS-NN:
```yaml
tests:
  ...
  sample.tensorflow.helloworld.cmsis_nn:
    tags: tensorflow
    platform_allow: mps3_an547
    extra_configs:
    - CONFIG_TENSORFLOW_LITE_MICRO_CMSIS_NN_KERNELS=y
```

Reply:
Done